config/defaults.go

```diff
@@ -21,7 +19,6 @@ var DefaultNodeConfig = NodeConfig{
 	Aggregator:     false,
 	LazyAggregator: false,
 	BlockManagerConfig: BlockManagerConfig{
-		BlockTime:   30 * time.Second,
```

Comment:
i would not remove this. set the default here to 1s and remove the re-init in manager.go.

Reply:
you can do the same for `DABlockTime`

Reply:
Done